dojo-docs/docs/dojo-api.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: default
+title: Dojo API
+has_toc: true
+---
+
+# The Dojo API
+
+This document outlines how to interact with the Dojo API to fetch models, execute models, and fetch model runs.
+
+## Model Discovery
+
+You can search Dojo for models using the `GET /models`. For example, to find models related to `crops` you would send:
+
+```
+curl -X 'GET' \
+  'https://api.dojo-modeling.com/models?query=crops&size=10' \
```

Comment:
Another reminder- possible pre-existing issue:

I think this url is old/wrong. Likely existing issue on dojo-modeling.com. I wonder if it is just an example and not meant to be tried out, but maybe we need a pro-tip kind-of note stating that all these urls are examples for when others deploy/self-host the app?

Another option is to update these to the correct urls where they are currently deployed at. It would have auth around them, but our existing transition users could copy them and use the endpoints seamlessly from the docs. Thoughts?